app/.env.example

```diff
@@ -2,7 +2,7 @@ PORT=<Port>
 HOST=<host address>
```

Comment:
is this whole file needed now that we have the configuration described in `config.ts`?